lib/fbe/octo.rb

```diff
@@ -75,6 +76,7 @@ def Fbe.octo(options: $options, global: $global, loog: $loog)
           methods: [:get],
           backoff_factor: 2
         )
+        builder.use(Fbe::FaradayMiddleware::Quota, logger: loog, pause: options.pause || 60)
```

Comment:
@Suban05 here, I would call it `github_api_pause` (to make it clear on the highest lever, what is this "pause" about)